package.json

```diff
@@ -145,6 +147,7 @@
     "install": "0.12.1",
     "intern": "4.2.2",
     "leadfoot": "1.7.4",
+    "npm": "^6.4.1",
```

Comment:
@g-k - do we need npm?

Reply:
We need a new enough version of npm to have the command `npm audit --json`, upgrading the node version would work too.

Reply:
Let me try specifying this in engines https://docs.npmjs.com/files/package.json#engines